### ISSUE TITLE
error with ATACAnnotateChromatinAccessibility

### ISSUE THREAD

**Issue by WWJiaxuan** [liulab-dfci/MAESTRO#134]
Hi,

Thanks for your useful package!

I encountered an error when running `ATACAnnotateChromatinAccessibility` as below:

> sh: bgzip: command not found
sort: write failed: standard output: Broken pipe
sort: write error
Not a BGZF file 'ATAC.GIGGLE/0.peaks.bed.gz'
giggle: Error loading query file ATAC.GIGGLE/0.peaks.bed.gz.
Error in read.table(paste0(outputBed, ".result.xls"), sep = "\t", row.names = NULL,  :
  no lines available in input
In addition: Warning message:
In system(cmd) : error in running command

May I know how I can address it? Thanks in advance and looking forward to your reply!

**Comment by WWJiaxuan** [comment]
Never mind, I guess I figured it out by installing `bgzip`.
